linux_os/guide/services/avahi/disable_avahi_group/service_avahi-daemon_disabled/tests/service_disabled.pass.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# packages = avahi
+# packages = {{{- ssgts_package("avahi") -}}}
```

Comment:
To actually make this work, I had to remove the '-'. So something like:
`# packages = {{{ ssgts_package("avahi") }}}`

Not truly sure why, since the '-' seems to be related to whitespace. Could it be a difference in jinja versions?

Reply:
@dodys Yes, `{{{- -}}}` is supposed to remove whitespace before/after the macro expansion. Otherwise, I'd get results like:

```
...
# packages =
avahi-daemon
...
```

(on two separate lines) and the test suite wouldn't run. What Jinja version are you using?

Reply:
2.10

Reply:
@dodys Did you ever get this working? I've gone ahead and rebased the PR. :-)

Reply:
only worked without the '-'
I haven't done anything else

Reply:
The dash at the end is wrong - it results in concatenation of the first line of the test scenario with the package list. Just removing that dash should work just fine.